### PR TITLE
Translation corrections

### PR DESCRIPTION
**Pull request by Tymaeus**
Various translation corrections.

### CODE REVIEW

File: Update/some_003.txt [hunk 1]
```diff
@@ -1943,7 +1943,7 @@ void main()
 			NULL, "\"......Oyashiro-sama...", Line_WaitForInput);
 	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_hatakeg1004", 256, TRUE);
 	OutputLine(NULL, "…しわけ…せん…」",
```
Comment (Tymaeus) [inline Update/some_003.txt:1945]: I am not 100% sure on this, but I believe that this is a fragmented version of 許しわけありません, as in Oyashiro-sama won't forgive those who leave Hinamizawa, tying back to Rena's story.

File: Update/some_003.txt [hunk 2]
```diff
@@ -1104,7 +1104,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300226", 256, TRUE);
 	OutputLine(NULL, "「夏美は出席番号が前のほうだから、たぶん来週頭には回ってくるよ。",
-			NULL, "\"Your attendance number is done the old way, so your career meeting will probably come around next week.", Line_WaitForInput);
+			NULL, "\"Your attendance number is on the earlier end, so your career meeting will probably come around next week.", Line_WaitForInput);
```
Comment (DoctorDiablo) [inline Update/some_003.txt:1107]:
```suggestion
			NULL, "\"Your attendance number is on the lower end, so your career meeting will probably come around next week.", Line_WaitForInput);
```

File: Update/some_003.txt [hunk 3]
```diff
@@ -1796,14 +1796,14 @@ void main()
...
-			NULL, "\"Oh no, is it already time for the broadcast to start?\"", Line_Normal);
+			NULL, "\"Oh, come to think of it, is it already time for the broadcast to start?\"", Line_Normal);
 	ClearMessage();
 
 //老人rvS04/00/VT1B_elderMd1006.「ワシゃ、ドラマの再放送の方がええんじゃがの～」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>老人</color>", NULL, "<color=#f5e6d3>Old person</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_eldermd1006", 256, TRUE);
 	OutputLine(NULL, "「ワシゃ、ドラマの再放送の方がええんじゃがの～」",
-			NULL, "\"It's a rerun of one of my dramas, isn't it~?\"", Line_Normal);
+			NULL, "\"I'd prefer a rerun of one of my dramas~\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/some_003.txt:1806]:
```suggestion
			NULL, "\"I'd prefer a rerun of one of my dramas~.\"", Line_Normal);
```

File: Update/some_003.txt [hunk 4]
```diff
@@ -1891,7 +1891,7 @@ void main()
 //rおじーちゃんは私に振り向きもしない。気づいていないはずはないんだけど、…ただじっと、ぼんやりと外を見ていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "おじーちゃんは私に振り向きもしない。気づいていないはずはないんだけど、…ただじっと、ぼんやりと外を見ていた。",
-			NULL, "The old man didn't turn toward me. He must not have noticed... He just continued gently looking outside.", Line_Normal);
+			NULL, "The old man didn't turn toward me. He should have noticed, ...but he just continued vacantly looking outside.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/some_003.txt:1894]:
```suggestion
			NULL, "The old man didn't turn toward me. He should have noticed... but he just continued vacantly looking outside.", Line_Normal);
```

**Pull request merged**